spec/fixtures/mocks/submission_with_file.json

```diff
@@ -12,29 +12,9 @@
       "order_total_value": 42.42,
       "purchase_order_number": null,
       "status": "completed",
-      "report_no_business?": false
-    },
-    "relationships": {
-      "files": {
-        "data": [
-          {
-            "type": "submission_files",
-            "id": "8d0e4289-5fde-4faa-b78e-922846d8460a"
-          }
-        ]
-      }
+      "report_no_business?": false,
+      "file_key": "12345",
```

Comment:
So there's no actual client-side attribute for this, it just shows up automagically on the object by dint of the fact it was here in the JSON?

Reply:
Yes, that's how JSONAPI works.